apps/infrastructure/src/sns_topic.tf

```diff
@@ -0,0 +1,10 @@
+resource "aws_sns_topic" "metric_alarm" {
+  name         = "cloudwatch_metric_alarms"
+  display_name = "DevPortal Alarms Topic"
+}
+
+resource "aws_sns_topic_subscription" "metric_alarm" {
+  protocol  = "email"
+  endpoint  = format("devportal-alerts+%s@pagopa.it", var.environment)
```

Comment:
I've decided that sending email for both environment, using the email alias, could help us to identify possible issues also when we work on dev environment.